scanner/updater/export.go

```diff
@@ -48,32 +48,34 @@ func Export(ctx context.Context, outputDir string) error {
 	defer func() {
 		closeErr := zstdWriter.Close()
 		if closeErr != nil {
-			zlog.Error(ctx).Err(closeErr).Msg("Failed to closing zstdWriter")
+			zlog.Error(ctx).Err(closeErr).Msg("Failed to close zstd writer")
 		}
 	}()
 
 	updaterSet, err := manual.UpdaterSet(ctx, nil)
 	if err != nil {
 		return err
 	}
-	outOfTree := [][]driver.Updater{
-		make([]driver.Updater, 0),
-	}
-	outOfTree = append(outOfTree, updaterSet.Updaters())
+	outOfTree := append(make([][]driver.Updater, 1), updaterSet.Updaters())
```

Comment:
Can you explain this? What is the purpose of the `make([][]driver.Updater, 1)`?

Reply:
It should be same as 
```
outOfTree := make([][]driver.Updater, 1)
outOfTree = append(outOfTree, updaterSet.Updaters()...)
```
Do you mean we should initialize the slice with 0 size?